### PR TITLE
SQL-1884: Make generate expansions silent

### PR DESCRIPTION
**Pull request by pmeredit**

### CODE REVIEW

File: evergreen.yml [hunk 1]
```diff
@@ -64,6 +64,7 @@ functions:
   "generate expansions":
     - command: shell.exec
       params:
+        silent: true
```
Comment (nbagnard) [inline evergreen.yml:67]: Why not adding `set +o xtrace` after `set -o errexit` like you did for the Beta branch?

Comment (pmeredit) [reply evergreen.yml:67]: xtrace won't matter when it's silent. I've abandoned the beta branch updates until the sign task is working here. I have silent: false right now for signing while I try to figure out why it's not working

**Comment by pmeredit** [comment]
Converting to draft to make sure it's clear that this is still actively being worked on (using the PR to run evergreen since it's slightly easier)

File: evergreen.yml [hunk 2]
```diff
@@ -398,11 +406,16 @@ task_groups:
...
   - name: windows-64
     display_name: "Windows"
-    run_on: [windows-64-vs2019-large]
     tasks:
       - name: build
+        run_on: [windows-64-vs2019-large]
       - name: test-integration-group
+        run_on: [windows-64-vs2019-large]
       - name: sign
+        run_on: [windows-64-vs2015-large]
```
Comment (terakilobyte) [inline evergreen.yml:415]: How onerous to have to add all the other things for this one change.

File: evergreen.yml (hunk 1 above)
Comment (terakilobyte) [reply evergreen.yml:67]: We can also use external scripts, but I think silent keeps is simpler.

**Review by terakilobyte — APPROVED** [review]

File: evergreen.yml (hunk 2 above)
Comment (nbagnard) [reply evergreen.yml:415]: Technically, we could have a different build variant for signing and leave this one alone for the other tasks.

**Review by nbagnard — APPROVED** [review]
LGTM

File: evergreen.yml [hunk 3]
```diff
@@ -225,10 +223,20 @@ functions:
       params:
         silent: true
         working_dir: mongo-powerbi-connector
+        env:
+          SELF_SIGNED_PWD: "${self_signed_pwd}"
+          SELF_SIGNED_CERT: "${self_signed_cert}"
+          SELF_SIGNED_KEY: "${self_signed_key}"
+          SELF_SIGNED_PFX: "${self_signed_pfx}"
```
Comment (bucaojit) [inline evergreen.yml:230]:
```suggestion
```
This was from the troubleshooting I recall, the pfx is generated now that things are working.

**Review by bucaojit — APPROVED** [review]
LGTM. 
One minor comment about the extra variable.

**Pull request merged**